src/pages/Upload.js

```diff
@@ -775,7 +787,7 @@ const UploadData = _.flow( // eslint-disable-line lodash-fp/no-single-compositio
 
   // Make sure we have a valid step once the workspaces have finished loading
   useEffect(() => {
-    if (!stepIsEnabled(currentStep) && !loadingWorkspaces) {
+    if (!stepIsEnabled(currentStep) && workspaces) {
```

Comment:
`useWorkspaces` initially returns `false` for `loadingWorkspaces`, causing this validation to happen before the workspaces list even begins loading.

Reply:
```suggestion
    if (!stepIsEnabled(currentStep) && !!workspaces) {
```